mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SkipInstallationChecker.java

```diff
@@ -0,0 +1,52 @@
+package org.arquillian.smart.testing.mvn.ext;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.arquillian.smart.testing.mvn.ext.MavenPropertyResolver.isSkipTestExecutionSet;
+import static org.arquillian.smart.testing.mvn.ext.MavenPropertyResolver.isSpecificTestClassSet;
+
+class SkipInstallationChecker {
+
+    private static final List<String> EXPECTED_GOALS = Arrays.asList(
+        new String[] {"test", "prepare-package", "package", "pre-integration-test", "integration-test",
+            "post-integration-test", "verify", "install", "deploy", "pre-site", "site", "post-site", "site-deploy"});
+    private final List<String> goals;
+    private String reason;
+
+    SkipInstallationChecker(List<String> goals) {
+
```

Comment:
White space?

Reply:
Yeah, because I love space. And white space in particular :-D
Good catch 👍